solutions/setup-dev-environment/README.md

```
@@ -51,8 +53,8 @@ To solve this, try increasing your `max_user_instances` and `max_user_watches`:
     ``` 
 * To permanently increase the limits
     ```
-    $ sudo echo "fs.inotify.max_user_watches=1024" >> /etc/sysctl.conf
-    $ sudo echo "fs.inotify.max_user_instances=1024" >> /etc/sysctl.conf
+    $ echo "fs.inotify.max_user_watches=1024" | sudo tee -a /etc/sysctl.conf
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)